scripts/src/java/org/oppia/android/scripts/proto/file_content_validation_checks.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Checks to ensure the usage of any prohibited file content in the codebase
+message FileContentChecks {
+  // List of all the file content checks
+  repeated FileContentCheck file_content_checks = 1;
+}
+
+// Check to ensure if any prohibited file content is used in the codebase
+message FileContentCheck {
+    // Matches to filename in which to do the content check.
```

Comment:
Please double check indentation. We should indent consistently, and in this case, proto files are 2-space indented.

Reply:
done.